MaxText/train.py

```diff
@@ -310,6 +311,14 @@ def record_goodput(recorder, config, step=None, job_start=False, job_end=False):
     if step is not None:
       recorder.record_step_start_time(step)
 
+def check_example_batch(config, example_batch):
+  if config.max_checkify:
+    jittable_f = checkify.checkify(
+        lambda x: checkify.check(jnp.any(x > -1), "Batch contains bad synthetic data!")
```

Comment:
sorry I'm not following this -- won't there be bad synthetic data but it will be in positions that should be clipped out in MaxText downstream?

Reply:
Fixed it